client/resteasy-client/src/main/java/com/cerner/beadledom/client/resteasy/ResteasyCorrelationIdContext.java

```diff
@@ -27,9 +27,13 @@
 
   @Override
   public String getCorrelationId() {
-    HttpHeaders headers = ResteasyProviderFactory.getContextData(HttpHeaders.class);
 
-    String correlationId;
+    String correlationId = com.cerner.beadledom.correlation.CorrelationIdContext.get();
```

Comment:
Where does the setting of correlationID happen? Is it on the server-side?

Reply:
Yes, server-side in the CorrelationIdFilter, https://github.com/cerner/beadledom/blob/8a7b63eb74bda01165675f0a39c69f30d5ab6f9d/jaxrs/src/main/java/com/cerner/beadledom/jaxrs/provider/CorrelationIdFilter.java#L82